### ISSUE TITLE
No response from the server when I ask any question.

### ISSUE THREAD

**Issue by aakashsardana** [snypiie/gitagpt#11]
I was trying to use Gita GPT but it seems there is no response from the server when I ask any questions.

After Investigation, I came to know that the `generate` API is returning 500.

Can someone please look into it?

**Comment by sanjeevkse** [comment]
Yes!! I too faced the issue. There is the response of 500 status code.

**Comment by Kushal997-das** [comment]
Guys if this one is `gitagpt` then what is this https://gitagpt-bollingen.streamlit.app/  ?

**Comment by Kushal997-das** [comment]
@snypiie ?

**Comment by sanjeevkse** [comment]
Handled the error response from server side, if all looks good. 
Kindly approve and merge it.
